README.md

```diff
@@ -24,7 +24,7 @@ $ npm install -g @bcgov/nrdk
 $ nrdk COMMAND
 running command...
 $ nrdk (-v|--version|version)
-@bcgov/nrdk/0.1.0-rc.28 darwin-x64 node-v12.14.1
+@bcgov/nrdk/0.1.0-rc.39 darwin-x64 node-v12.14.1
```

Comment:
Can you please add description to preferably all 3 commands nrdk clean, deploy and on:jira:issue but atleast to the last one

Reply:
the README file is auto generated/updated by oclif